### PR TITLE
chore: operations WorkStream FAQ additions

### PR DESCRIPTION
**Pull request by Neverwas-dev**
### Description

---

This is the initial push for the newly created FAQ section for the DAO repo and GitBook.

### Notice

---

- [x] Have you followed the guidelines in our Contributing guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

### Pull Request Type

---

 - [ ] 🐛 Bug fix (Non-breaking Change: Fixes an issue)
 - [x] 🛠️ Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ]  💅 New Feature (Breaking/Non-breaking Change)

### Notes

---

This was created in a few iterations on GitBook, I cleaned up most of the markdown trivial warnings.

### Risk

---

As of now just adding the structure and folders with the new files, not let linking from the root dir.

### CODE REVIEW

**Comment by Neverwas-dev** [comment]
Forgive the deletions from docs the GitBook was a little janky with different repos and had different root structure, mashed it together after the fact. I would have preferred just cloning docs and make the changes but we had multiple GitBook trials and team members so it worked out this way. All the original Doc repo files should be intact except for .gitbook.yaml which has a few added lines for the FAQ folder structure.

**Comment by Neverwas-dev** [comment]
Noticed after the fact that the assets folder was missing, and mistyped it in uppercase when adding it back, with just images in the folder I don't remember how to force rename it on github. VScode is displaying it correctly.

File: faq/wallets/keepkey/keepkey-terms-of-service.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+KeepKey has the absolute right to remove and/or delete without notice any User Content within its control that it deems objectionable. You consent to such removal and/or deletion and waive any claim against KeepKey for such removal and/or deletion. KeepKey is not responsible or liable for failure to store posted content or other materials you transmit through the Service. You should take measures to preserve copies of any data, material, content or information you post on the Service or any other sites or platforms.
+
+1. Our Proprietary Rights
```
Comment (0xApotheosis) [inline faq/wallets/keepkey/keepkey-terms-of-service.md:33]: The numbering from here to "Limitation of Liability" is all "1.".

If there were in a valid list they'd increment, but they are not due to the way this is formatted.

I expect the rendered result is not what we expect.

File: faq/app.shapeshift/blockchains.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+{% embed url="https://ethereum.org/en/" %}
+
+### Bitcoin
+
+[Bitcoin](https://www.investopedia.com/terms/b/bitcoin.asp) is a [cryptocurrency](https://www.investopedia.com/terms/c/cryptocurrency.asp), a virtual currency designed to act as money and a form of payment outside the control of any one person, group, or entity, and thus removing the need for third-party involvement in financial transactions. It is rewarded to blockchain miners for the work done to verify transactions and can be purchased on several exchanges.
+
+{% embed url="https://www.bitcoin.com/" %}
+
+### DOGECOIN
```
Comment (0xApotheosis) [inline faq/app.shapeshift/blockchains.md:23]: Why caps?

File: faq/app.shapeshift/blockchains.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+“BNB stands for “Build the community, and let the community build, build, ’n build.”
+
+The recently released BNB Chain comprises the BNB Beacon Chain for BNB Chain governance (i.e., staking and voting) and the BNB Smart Chain, which remains [Ethereum Virtual Machine (EVM) compatible](https://cointelegraph.com/ethereum-for-beginners/architectural-components-of-the-ethereum-blockchain-what-are-they).
+
+The BNB Chain aims to increase interoperability and develop the underlying infrastructure of the “world’s parallel virtual environment,” a step forward in Binance’s recent MetaFi efforts.
+
+{% embed url="https://www.bnbchain.org/en" %}
+
+### OSMOSIS
```
Comment (0xApotheosis) [inline faq/app.shapeshift/blockchains.md:71]: Why caps?

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]
Looks reasonable to me!

- Can we undo the rename of the `assets` directory to `ASSETS`? Unless this is a GitBook thing there is no reason to do this.
- `ASSETS/Bounty Ticket Flow Diagram.drawio.png` - don's use spaces in file names 🙏 
- The filenames in `.gitbook/assets` could use some love

File: faq/app.shapeshift/blockchains.md (hunk 3 above)
Comment (Neverwas-dev) [reply faq/app.shapeshift/blockchains.md:71]: I wasn't paying attention. In vscode the folder is correct it won't recognize the change and I did a push just to verify. What's the easiest way to refresh this my git console commands are rusty

Comment (0xApotheosis) [reply faq/app.shapeshift/blockchains.md]: If you rename the folder and then in a terminal check that the changes are registered with `git status`.

Then add them either explicitly by file name with `git add <path>`, or add all changes `git add *`.

I can't push to your repo, but:

Screenshot 2022-11-11 at 2 06 13 pm

Comment (0xApotheosis) [reply faq/app.shapeshift/blockchains.md]: You'll then need to resolve merge conflicts.

**Comment by 0xApotheosis** [comment]
Closing in favour of https://github.com/shapeshift/docs/pull/48.

**Pull request closed without merging**